docs/reference/go/v0/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Nitric's Go SDK
+description: Using Nitric with Go
+---
+
+This SDK reference provides documentation for the functions and methods in Nitric's GO library.
+
+The library provides the ability to define and interact with cloud resources, as well as build application logic like functions/handlers.
+
+## Installation
+
+If you used a Nitric starter template for Go to scaffold your project, then `github.com/nitrictech/go-sdk` will already be included in the dependencies in your `go.mod` file. Using starter templates is the recommended installation option since it ensures the other files and configuration needed are also set up.
+
```

Comment:
Suggest a shorter sentence here - The Nitric starter template for Go includes the Go SDK module dependency `github.com/nitrictech/go-sdk` within your `go.mod` file.